### PR TITLE
Apple Silicon support statement [no JIRA]

### PR DESCRIPTION
**Pull request by aednichols**

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]

**Review by cjllanwarne — CHANGES_REQUESTED** [review]
IMO the text of this statement belongs in readthedocs, but the changelog can still point at it

**Comment by cjllanwarne** [comment]
(👍  and thank you for the documentation content itself though!)

**Review by cjllanwarne — APPROVED** [review]
LGTM if you can resolve the merge conflict

File: docs/Releases.md [hunk 1]
```diff
@@ -20,4 +20,20 @@ This is the main artifact in Cromwell releases that contains all executable Crom
 
 [Java 8](http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html) is required to run Cromwell.
 
-For users running a cromwell server [a docker image](https://hub.docker.com/r/broadinstitute/cromwell) has been made available.
+For users running a Cromwell server [a docker image](https://hub.docker.com/r/broadinstitute/cromwell) has been made available.
+
+### Apple Silicon support statement (updated 11/20)
```
Comment (mcovarr) [inline docs/Releases.md:25]:
```suggestion
### Apple Silicon support statement (updated 2020-11-17)
```

**Comment by geoffjentry** [comment]
Cromwell on my iPhone, oh yeah

**Pull request merged**